docs/compute-element/htcondor-ce-overview.md

```diff
@@ -29,11 +21,7 @@ What is HTCondor-CE?
 
 HTCondor-CE is a special configuration of the HTCondor software designed to be a job gateway solution for the OSG. It is configured to use the [JobRouter daemon](http://research.cs.wisc.edu/htcondor/manual/v8.6/5_4HTCondor_Job.html) to delegate jobs by transforming and submitting them to the site’s batch system.
 
-### How is HTCondor-CE different from a GRAM CE?
-
-The biggest difference you will see between an HTCondor-CE and a GRAM CE is in the way that jobs are submitted to your batch system; HTCondor-CE uses the built-in JobRouter daemon whereas GRAM CE uses jobmanager scripts written in Perl. Customizing your site’s CE now requires editing configuration files instead of editing jobmanager scripts.
-
-Listed below are some other benefits to switching to HTCondor-CE:
+Other benefits of running the HTCondor-CE:
```

Comment:
Do we need this section at all? Benefits of running the HTCondor-CE = your site can join the OSG.

If you think we still do, remove the word "Other" because no benefits are listed above.

Reply:
I assume that in the (hopefully) near future, the docs talk a lot more about the hosted CE setup.  When it does, I was hoping this would be much more applicable.

Anyhow, I agree about removing the word "other".

Reply:
Fair enough.